start_bundle_coupled.R

```diff
@@ -474,7 +477,12 @@ for(scen in common){
     if (i == start_iter_first && ! start_now && all(file.exists(cfg_rem$files2export$start[path_gdx_list]) | unlist(gdx_na))) {
         start_now <- TRUE
     }
-
```

Comment:
delete folders for runs that it wants to start. This can happen only if these folders could not be used to continue the scenario from there, for some example they don't contain fulldata.gdx or mif file